components/scream/src/physics/spa/tests/spa_main_test.cpp

```diff
@@ -366,7 +366,15 @@ TEST_CASE("spa_read_data","spa")
       auto ssv = [&](const view_3d<Spack>& v, const int dof_i, const int n) -> col_type {
         return ekat::subview(ekat::scalarize(v),dof_i,n);
       };
+      check_bounds (ssv(aer_g_sw_h,dof_i,n),ssv(aer_g_sw_beg,dof_i,n));
       check_bounds (ssv(aer_ssa_sw_h,dof_i,n),ssv(aer_ssa_sw_beg,dof_i,n));
+      check_bounds (ssv(aer_tau_sw_h,dof_i,n),ssv(aer_tau_sw_beg,dof_i,n));
+    }
+    for (int n=0;n<nlwbands;n++) {
+      auto ssv = [&](const view_3d<Spack>& v, const int dof_i, const int n) -> col_type {
```

Comment:
Since you added dof_i and n in the lambda signature, you could move the definition out of the dof_i loop, so you create one lambda only once.